api/src/contracttest/java/org/pmiops/workbench/consumer/AppsApiTest.java

```diff
@@ -290,7 +285,12 @@ RequestResponsePact listAppsByProject(PactDslWithProvider builder) {
         .willRespondWith()
         .status(200)
         .headers(contentTypeJsonHeader)
-        .body(newJsonArray(array -> {}).build())
+        .body(
+            newJsonArray(
+                    apps -> {
+                      apps.object(this::applyAppExpectations);
+                    })
+                .build())
```

Comment:
nit: Don't need the curly braces 
```suggestion
        .body(newJsonArray(apps -> apps.object(this::applyAppExpectations)).build())
```